docs/user-guide/systemrequirements-zos.md

```diff
@@ -25,6 +25,20 @@ Be sure your z/OS system meets the following prerequisites:
 
 - zFS volume has at least 1200 mb of free space for Zowe server components, the corresponding keystore, instance configuration files and logs, and third-party plug-ins.
 
+- SDSF
+
+  During the installation of Zowe, there are a few operator commands issued to SDSF to know the status of the job submitted to ZOS. Hence it is expected to have SDSF installed on ZOS, otheriwse user can expect a below error message.
```

Comment:
Please rephrase to state that it's used to interface with JES and used to send MVS commands (such as the start command for the Zowe Started Task)